### PR TITLE
Document the boot.properties behavior in the README

### PR DESCRIPTION
**Pull request by CarolynRountree**
Document that WDT creates a boot.properties with credentials for development domains

### CODE REVIEW

File: site/create.md [hunk 1]
```diff
@@ -30,3 +30,24 @@ If the model or variables file contains encrypted passwords, add the `-use_encry
 ### Using Multiple Models
 
 The Create Domain Tool supports the use of multiple models, as described in [Using Multiple Models](../README.md#using-multiple-models).
+
+### Development Domain and "boot.properties"
```
Comment (rosemarymarano) [inline site/create.md:34]: Development Domain and "boot.properties" -> Development domain and `boot.properties` file  (code font for `boot.properties`, not quotation marks)

File: site/create.md [hunk 2]
```diff
@@ -30,3 +30,24 @@ If the model or variables file contains encrypted passwords, add the `-use_encry
...
+
+When creating a development domain, WDT provides the convenience of creating a boot.properties for each of the servers in the domain. The boot.properties will contain encrypted values of the Admin Server username and password. When the Admin server or Managed server is started, WebLogic Server will bypass the prompt for credentials, instead using the credentials from the boot.properties.
```
Comment (rosemarymarano) [inline site/create.md:36]: boot.properties -> `boot.properties` file (please change `boot.properties` to `boot.properties` file, globally)
Admin Server username -> Administration Server user name
Admin server or Managed server -> Administration Server or Managed Server
credentials, instead using the -> credentials, and instead use the

File: site/create.md [hunk 3]
```diff
@@ -30,3 +30,24 @@ If the model or variables file contains encrypted passwords, add the `-use_encry
...
+
+A domain is considered production mode if the ServerStartMode option is set to 'prod' and/or the domain ProductionModeEnabled is set to true. The default value for both of these attributes is development.
```
Comment (rosemarymarano) [inline site/create.md:38]: A domain is considered production mode if the ServerStartMode option is set to 'prod' and/or the domain ProductionModeEnabled is set to true. -> A domain is in production mode if the `ServerStartMode` option is set to `prod` or the domain `ProductionModeEnabled` option is set to `true`.  (Use back ticks to render code font)
development. -> development mode.

File: site/create.md [hunk 4]
```diff
@@ -30,3 +30,24 @@ If the model or variables file contains encrypted passwords, add the `-use_encry
...
+
+The boot.properties is stored in the domain home on the machine where WDT runs. It is stored for each server as <domain_home>/servers/<server_name>/security/boot.properties.
```
Comment (rosemarymarano) [inline site/create.md:40]: <domain_home>/servers/<server_name>/security/boot.properties -> `<domain_home>/servers/<server_name>/security/boot.properties`  (File paths should be in code font)

File: site/create.md [hunk 5]
```diff
@@ -30,3 +30,24 @@ If the model or variables file contains encrypted passwords, add the `-use_encry
...
+
+The following is a model example with both attributes explicitly set to development.
```
Comment (rosemarymarano) [inline site/create.md:42]: development. -> development mode.

File: site/create.md (hunk 2 above)
Comment (CarolynRountree) [reply site/create.md]: Done

File: site/create.md (hunk 4 above)
Comment (CarolynRountree) [reply site/create.md]: Done.

File: site/create.md (hunk 3 above)
Comment (CarolynRountree) [reply site/create.md]: Done

**Pull request merged**